docker/sane-pdf-reports/get_build_artifacts.py

```diff
@@ -35,11 +37,12 @@ def get_args():
     return vars(parser.parse_args())
 
 
-def send_request(url, params):
-    headers = {'Accept': 'application/json'}
+def send_request(url):
+    headers = {
+        'Accept': 'application/json',
+        'Circle-Token': os.getenv(ENV_TOKEN_KEY, "")
```

Comment:
do we need token here? sane reports is public as well as circle artifacts
![image](https://user-images.githubusercontent.com/18641362/84790626-3be88380-affa-11ea-9868-8cca166e4ba7.png)

Reply:
Nope :)
But if we will need it will work.